package.json

```diff
@@ -53,11 +53,11 @@
     "webpack-config-single-spa-react-ts": "^4.0.0",
     "webpack-config-single-spa-ts": "^4.0.0",
     "webpack-dev-server": "^4.0.0",
-    "webpack-merge": "^5.8.0"
+    "webpack-merge": "^5.8.0",
+    "axios": "^1.6.7"
```

Comment:
I think axios should be in `dependencies`

Reply:
@adongare I moved it to dev because it's only used for tests, do you want me to move it back to dependencies?

Reply:
I think it is used in a couple of services e.g. useOrganizationApi